src/steamship/agents/react/react.py

```diff
@@ -54,6 +54,9 @@ class ReACTAgent(LLMAgent):
 Previous conversation history:
 {chat_history}
 
+Other relevant previous conversation:
```

Comment:
do we expect that we will need both at the same time?  I can't convince myself that we need both `Previous conversation history:` and `Other relevant previous conversation:` to be optional somehow. When empty, they don't contribute too many tokens, I suppose.

Reply:
I was imagining a long conversation with an agent.  The Previous conversation history might include the most recent N messages, and the 'other relevant' bit might include a few snippets that most closely matched the last input.  

I'm very open to any other / better suggestions on how to incorporate this into the prompt.